homeassistant/components/group/config_flow.py

```diff
@@ -30,13 +32,27 @@ def basic_group_config_schema(domain: str) -> vol.Schema:
     )
 
 
+BINARY_SENSOR_OPTIONS_SCHEMA = basic_group_options_schema("binary_sensor").extend(
+    {
+        vol.Required(CONF_ALL, default=False): selector.selector({"boolean": {}}),
+        vol.Optional(CONF_DEVICE_CLASS): selector.selector(
+            {"select": {"options": list(map(str, BinarySensorDeviceClass))}}
+        ),
```

Comment:
Frontend currently doesn't allow unselecting a selected option.
We can work around this in backend by adding a "None" or "-" option, or update frontend to allow unselecting the option for optional select selectors.

Reply:
This needs to be fixed in the frontend.

Reply:
As agreed we inject a "none" option instead :)